src/index.html

```diff
@@ -1,3 +1,4 @@
+<!-- #region head -->
 <!doctype html>
 <html lang="en">
```

Comment:
The `<!DOCTYPE html>` declaration should be in lowercase to adhere to the HTML5 specification and maintain consistency with the code style guide.